pyaedt/generic/DataHandlers.py

```diff
@@ -55,13 +55,17 @@ def _tuple2dict(t, d):
 
 @pyaedt_function_handler()
 def _dict2arg(d, arg_out):
-    """
+    """Create a valid string of name-value pairs for the native AEDT API.
+
+    Prepend the argument string in `arg_aut` using the dict `d`
```

Comment:
```suggestion
    Prepend the argument string in ``arg_aut`` using the dict ``d``.
```
A single backtick renders the contents in italics rather than a literal string.

Reply:
Done. Also changed `arg_aut` to `arg_out`.